src/main/java/core/basesyntax/dao/ReadFileDaoImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.dao;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
+public class ReadFileDaoImpl implements ReadFileDao {
```

Comment:
DAO shouldn't be in the name
Class name should be/end with a noun - FileReaderImpl